src/v3/test/integration/authenticator-verification-email.test.tsx

```diff
@@ -52,6 +56,15 @@ describe('Email authenticator verification when email magic link = undefined', (
     });
 
     it('renders the initial form with resend alert box', async () => {
+      const startTimestamp = mockSystemTime;
+      jest.spyOn(global, 'sessionStorage', 'get').mockReturnValue({
+        length: 0,
+        clear: () => jest.fn(),
+        getItem: () => startTimestamp.toString(),
+        setItem: () => jest.fn(),
+        key: () => null,
+        removeItem: () => jest.fn(),
+      });
```

Comment:
Instead of repeating the `spyOn` you can define this once in the describe, then change the mockReturnValue in the beforeEach. Since it looks like the implementation is the same in each test, no need to override the mockReturnValue for the session in each individual test.

i.e.
```
describe('', () => {
   ...
   const sessionStorageSpy = jest.spyOn(global, 'sessionStorage', 'get');
   beforeEach(() => {
      mockSystemTime = Date.now();
      sessionStorageSpy.mockReturnValue({
           length: 0,
           clear: () => jest.fn(),
           getItem: () => mockSystemTime.toString(),
           setItem: () => jest.fn(),
           key: () => null,
           removeItem: () => jest.fn(),
         });
      ...
   });
   
   it('some test', () => {
      ...
   });
});
```